### PR TITLE
Updating SES integration documentation (JSC#SOC-11140)

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/depl_nodes.xml [hunk 1]
```diff
@@ -7640,6 +7640,28 @@ radosgw_urls:
        No special configuration is needed for &o_comp;. It uses the
        configuration from the &o_blockstore; &barcl; to connect with volumes
        stored in SES.
+       For ephemeral storage, &o_comp; re-uses the <literal>rbd_store_user</literal>
+       and key from &o_blockstore; but has a separate <literal>rbd_store_pool</literal>
```
Comment (skazi0) [inline xml/depl_nodes.xml]: `key` should also be a literal

File: xml/depl_nodes.xml [hunk 2]
```diff
@@ -7640,6 +7640,28 @@ radosgw_urls:
...
+       defined in the SES configuration.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>RadosGW Integration</term>
```
Comment (skazi0) [inline xml/depl_nodes.xml]: Ardana docs use "RADOS Gateway" term here. Maybe we should do the same.

File: xml/depl_nodes.xml [hunk 3]
```diff
@@ -7640,6 +7640,28 @@ radosgw_urls:
...
+     <listitem>
+      <para>
+       Besides &o_blockstore;, SES cluster can also be used as a &o_objstore;
```
Comment (skazi0) [inline xml/depl_nodes.xml]: &o_blockstore; will translate to Cinder, right? If so, I think it's better to say "block storage" which is a bit more abstract and matches "object storage" at the end of this sentence.

File: xml/depl_nodes.xml [hunk 4]
```diff
@@ -7640,6 +7640,28 @@ radosgw_urls:
...
+       replacement for object storage. If <literal>radosgw_urls</literal> section is present
+       in uploaded SES configuration, first of the URLs are registered
```
Comment (skazi0) [inline xml/depl_nodes.xml]: "is registered"?

File: xml/depl_nodes.xml (hunk 3 above)
Comment (asettle) [reply xml/depl_nodes.xml]: https://github.com/SUSE-Cloud/doc-cloud/blob/maintenance/cloud_9/xml/entity-decl.ent#L270

Does indeed. At the end it translates to swift: https://github.com/SUSE-Cloud/doc-cloud/blob/maintenance/cloud_9/xml/entity-decl.ent#L95 

So it currently does match, but if you'd prefer the change I can do that.

File: xml/depl_nodes.xml [hunk 5]
```diff
@@ -7640,6 +7640,28 @@ radosgw_urls:
...
+       in the &o_ident; catalog as "&swift;"/"object-store" service. Some
```
Comment (skazi0) [inline xml/depl_nodes.xml]: what's the difference between `&swift;` and `&o_objstore;`?

Comment (asettle) [reply xml/depl_nodes.xml]: You can see all of the entitys here: https://github.com/SUSE-Cloud/doc-cloud/blob/maintenance/cloud_9/xml/entity-decl.ent#L76

File: xml/installation-installation-ses_integration.xml [hunk 6]
```diff
@@ -63,9 +63,9 @@ glance:
   rbd_store_user: glance
 nova:
   rbd_store_pool: cloud-vms
-  radosgw_urls:
-     - http://10.84.56.7:80/swift/v1
-     - http://10.84.56.8:80/swift/v1 </screen>
+radosgw_urls:
+  - http://10.84.56.7:80/swift/v1
+  - http://10.84.56.8:80/swift/v1 </screen>
```
Comment (skazi0) [inline xml/installation-installation-ses_integration.xml]: the second example has this list indented too much too.

File: xml/depl_nodes.xml (hunk 3 above)
Comment (skazi0) [reply xml/depl_nodes.xml]: I meant the literal "object storage" we have at the end of the sentence on following line.

File: xml/depl_nodes.xml (hunk 5 above)
Comment (skazi0) [reply xml/depl_nodes.xml]: thx!

File: xml/depl_nodes.xml (hunk 3 above)
Comment (asettle) [reply xml/depl_nodes.xml]: I see. I can change that.

**Comment by jeremy-moffitt** [comment]
superseded by https://github.com/SUSE-Cloud/doc-cloud/pull/1259

**Pull request closed without merging**